src/test/java/athena/ParserTest.java

```diff
@@ -34,14 +34,14 @@ public void setUp() {
     }
 
     /**
-     * Checks if the help command is shown if the user enters an invalid command.
-     *
-     * @throws CommandException Exception thrown if there is an error with the user entered command
+     * Checks if an exception is thrown if the user enters an invalid command.
      */
     @Test
-    public void parse_unknownCommandWord_returnsHelp() throws CommandException {
+    public void parse_unknownCommandWord_throwsException() {
         final String input = "unknown arguments";
-        parseAndAssertCommandType(input, HelpCommand.class);
+        assertThrows(CommandException.class, () -> {
```

Comment:
I think that for this one, it would be good to catch the `InvalidCommandException`, to ensure it is this specific exception that is being thrown.